Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -402,9 +477,7 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 var ignoredPaths = GetProjectProperty(NodejsConstants.AnalysisIgnoredDirectories);
 
                 if (!string.IsNullOrWhiteSpace(ignoredPaths)) {
-                    _analysisIgnoredDirs = ignoredPaths.Split(';').Select(x => '\\' + x + '\\').ToArray();
-                } else {
-                    _analysisIgnoredDirs = new string[0];
+                    _analysisIgnoredDirs.Append(ignoredPaths.Split(';').Select(x => '\\' + x + '\\').ToArray());

```

Comment:
It gets called on project load, and the ProjectNodes don't get reused on Reload, so it's effectively set.